library/Vanilla/Embeds/GettyEmbed.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0
+ */
+
+namespace Vanilla\Embeds;
+
+use Exception;
+/**
+ * Getty Embed.
+ */
+class GettyEmbed extends Embed {
+
+    /** @inheritdoc */
+    protected $domains = ['gettyimages.ca', 'gty.im'];
```

Comment:
What about "gettimages.com"? They actually seem to have `gettyimages.{}` for a ton of country specific domains.

![image](https://user-images.githubusercontent.com/1770056/41793008-c0a5ffa2-7628-11e8-9452-3667b3927e6d.png)

I don't think we care that much about the rest of them, but we at least need them the `.com` domain.

Reply:
Don't forget the tests.